### DISCUSSION TITLE
Idea: Environment Variable to force use of Passphrase Lockdown

### DISCUSSION THREAD

**Discussion by rajtsic** [pglombardo/PasswordPusher#1451] [Ideas]
Idea:
Make a Environment Variable to force use of Passphrase Lockdown - before you could "Push It!"

This is so that you need somekind of Passphrase Lockdown before you could push and send it.
This is to increase security.

**Comment by pglombardo** [comment]
Hi @rajtsic - late update but this is a good request.  I'd like to add this ability when I add support for policies.  Thanks for pointing it out.
